pkg/gds/global/v1/interface.go

```diff
@@ -0,0 +1,20 @@
+package global
```

Comment:
These interfaces generalize Object interactions with the database so that it's not leveldb specific.